doc/Language/variables.rakudoc

```diff
@@ -1340,7 +1340,7 @@ Module-Foo/
 =end code
 
 Additionally, a relative path (starting from the root resources directory of a
-distribution) to a file is specified under C<"resources"> field in the
+distribution) to a file must be specified under C<"resources"> field in the
```

Comment:
I think the original text reads better, because nothing `MUST` be under the `resources/` field (its optional to list anything)

Reply:
So done. 

BUT, should there be a note as to why it's even used? Is there a possibility it might be used someday? 

Also, is any mismatch caught by anything other than possibly App::Mi6?